src/components/AllWorkOrders.js

```diff
@@ -0,0 +1,193 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faTruck, faMapMarkerAlt } from '@fortawesome/free-solid-svg-icons'
+
+import api from '../queries/api'
+import { workOrderFields } from '../queries/fields'
+import { signalsWorkOrderStatuses } from '../constants/statuses'
+
+const fields = workOrderFields.baseFields
+const statuses = signalsWorkOrderStatuses
+
+class AllWorkOrders extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            allWorkOrdersData: [],
+            location: '',
+            currentPage: 1,
+            lastPage: 1,
+        }
+    }
+
+    componentDidMount() {
+        api.allWorkOrders()
+            .getAll()
+            .then(res => {
+                this.setState({
+                    allWorkOrdersData: res.data.records,
+                    lastPage: res.data.total_pages,
+                })
+                console.log(res.data)
+            })
+    }
+
+    handleChange = event => {
+        this.setState({
+            location: event.target.value,
+        })
+    }
+
+    handleSubmit = event => {
```

Comment:
Small idea here, but maybe we call this function `handleSearch`?  Also, when I opened the preview app, I noticed a small improvement we could make. Could we have some kind of loading state. So at the beginning we setState to loading (which renders a spinner in the search button) and then when the API call returns successful, we go back to the normal state.

Reply:
handleSearch is definitely more appropriate for this function 👍 - changed. For the spinner, I mirrored the from MyWorkOrders.js and also set state.allWorkOrdersData to [ ] upon submitting the search so that it clears results while showing the spinner.